common/src/main/java/com/tencent/rss/common/rpc/GrpcServer.java

```diff
@@ -63,14 +63,14 @@ public GrpcServer(RssBaseConf conf, BindableService service, GRPCMetrics grpcMet
           .forPort(port)
           .addService(ServerInterceptors.intercept(service, monitoringInterceptor))
           .executor(pool)
-          .maxInboundMessageSize(maxInboundMessageSize)
+          .maxInboundMessageSize((int)maxInboundMessageSize)
           .build();
     } else {
       this.server = ServerBuilder
           .forPort(port)
           .addService(service)
           .executor(pool)
-          .maxInboundMessageSize(maxInboundMessageSize)
+          .maxInboundMessageSize((int)maxInboundMessageSize)
```

Comment:
we should check `0 < maxInboundMessageSize < Integer.Max_VALUE`

Reply:
fixed by checkValue